### PR TITLE
Chore: Add Community guidelines for github repository

### PR DESCRIPTION
**Pull request by satyamyadav**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+`master` or development branch in the repository.
+
+## Request Features
+
+New feature requests are welcome. Analyse whether the idea fits within scope of
+the project. Then, detail your request, ensuring context and use case is provided.
+
+**Please provide:**
+
+* A detailed description the advantages of your request
```
Comment (aditya-kumawat) [inline CONTRIBUTING.md]: A detailed description of the advantages of your request

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+ 
+### Check If It's Been Fixed
+
+Check if the issue has been fixed — try to reproduce it using the latest
+`master` or development branch in the repository.
+
+## Request Features
+
+New feature requests are welcome. Analyse whether the idea fits within scope of
+the project. Then, detail your request, ensuring context and use case is provided.
```
Comment (aditya-kumawat) [inline CONTRIBUTING.md]: New feature requests are welcomed. Analyse whether the idea fits within the scope of project. If yes, detail your request while ensuring context and potential use cases.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+#### Before Commit
+
+Before committing, **you must ensure there are no linting errors and
+all tests pass.**
+
+To do this, run all tests:
+
+```bash
+npm test
+```
```
Comment (aditya-kumawat) [inline CONTRIBUTING.md]: To ensure the above conditions, run:

1. For lint:
```bash
npm lint
```
2. For tests:
```bash
npm test
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+We follow the [conventional commit messages](https://conventionalcommits.org/)
+convention in order to automate CHANGELOG generation and to automate
+semantic versioning.
+
+For example:
+
+* `feat: A new feature`
+* `fix: A bug fix`
+
+A commit of the type feat introduces a new feature to the codebase
```
Comment (aditya-kumawat) [inline CONTRIBUTING.md]: A commit of the type **feat** introduces a new feature to the codebase

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+(this correlates with MINOR in semantic versioning).
+
+e.g.:
+
+```
+feat: xxxxxxxxxx
+```
+
+A commit of the type fix patches a bug in your codebase (this correlates with PATCH in semantic versioning).
```
Comment (aditya-kumawat) [inline CONTRIBUTING.md]: A commit of the type **fix** patches a bug in your codebase (this correlates with PATCH in semantic versioning).

File: CONTRIBUTING.md (hunk 2 above)
Comment (aditya-kumawat) [reply CONTRIBUTING.md]: Then, detail your request, ensuring context and use case is provided.

I think this one needs update too. Multiple commas are not required.

**Review by aregee — APPROVED** [review]

**Pull request merged**